src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc

```diff
@@ -71,6 +72,9 @@ typedef SharedPtr<AlertManeuverRequest> CommandPtr;
 class AlertManeuverRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
+  AlertManeuverRequestTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock()) {}
```

Comment:
@dev-gh I would recommend to make expectations clean up in base class. Not in every RPC.